### PR TITLE
Final Version for Spot Tile Tests 

### PR DESCRIPTION
**Pull request by fabio-weareadaptive**
This is final version for the Spot Tile Tests written on the revamped Playwright framework.

### CODE REVIEW

File: packages/client/playwright/components/spotTile.component.ts [hunk 1]
```diff
@@ -24,10 +25,19 @@ export default class SpotTileComponent extends BaseComponent {
...
   }
 
+  public getToggle() {
+    return this.page.getByTestId("toggleButton")
+  }
+
+  public async getTileSate() {
+    const tileState = await this.page.evaluate(() => {
+      return window.localStorage.getItem("selectedView")
+    })
```
Comment (stephan-thibodeau) [inline packages/client/playwright/components/spotTile.component.ts:35]: I may be wrong but I think you only need to return `window.window.localStorage.getItem("selectedView")`

File: packages/client/playwright/tests/spot-tile.spec.ts [hunk 2]
```diff
@@ -133,4 +133,34 @@ test.describe("FX purchase", () => {
       await expect(eurusdTile.getMaxExceedError).not.toBeVisible()
     })
   })
+  test.describe("Toggle between prices and graph views", () => {
+    test("When I click the graph icon on the Live Rates bar then I should toggle from graph to price views", async ({}) => {
```
Comment (stephan-thibodeau) [inline packages/client/playwright/tests/spot-tile.spec.ts:137]: 💯

File: packages/client/playwright/components/spotTile.component.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { Locator, Page } from "@playwright/test"
 
 import { BaseComponent } from "../Base.component"
+import { GrReturn } from "react-icons/all"
```
Comment (stephan-thibodeau) [inline packages/client/playwright/components/spotTile.component.ts:4]: This import is not needed

**Review by stephan-thibodeau — CHANGES_REQUESTED** [review]

File: packages/client/playwright/components/spotTile.component.ts (hunk 3 above)
Comment (fabio-weareadaptive) [reply packages/client/playwright/components/spotTile.component.ts:4]: Removed ✅

File: packages/client/playwright/components/spotTile.component.ts (hunk 1 above)
Comment (fabio-weareadaptive) [reply packages/client/playwright/components/spotTile.component.ts:34]: You're right, I've simplified it accordingly to keep it cleaner 👌🏼

**Review by stephan-thibodeau — APPROVED** [review]

**Pull request merged**